vector/src/main/java/im/vector/view/PillView.java

```diff
@@ -91,13 +93,14 @@ private void initView() {
     /**
      * Tells if a pill can be displayed for this url.
      *
+     * @param supportedHosts the supported hosts for the Tchap links.
      * @param url the url
      * @return true if a pill can be made.
      */
-    public static boolean isPillable(String url) {
-        String linkedUrl = PermalinkUtils.getLinkedId(url);
-
-        return (null != linkedUrl) && (MXPatterns.isRoomAlias(linkedUrl) || MXPatterns.isUserId(linkedUrl));
+    public static boolean isPillable(@NonNull String[] supportedHosts, String url) {
+        return (null != url)
```

Comment:
`null != url` was not tested before, just as a remark

Reply:
this was tested by `getLinkedId()`